src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/ResultsController.cs

```diff
@@ -78,8 +78,10 @@ public ActionResult RenderResults(int? pageIndex, int weekNumber = 1)
                     })
                     .ToList();
 
+            var totalPages = (int)Math.Ceiling(this._resultsService.GetTotalTestResultsCount(weekNumber) / (double)pageSize);
```

Comment:
pageSize should not be double

Reply:
effectively, it doesn't matter, because Math.Ceiling gets the param of type double or decimal